src/boxed.tsx

```diff
@@ -21,6 +21,10 @@ type Props = {
     height?: number | string;
 };
 
+type InternalProps = {
+    borderRadius: 8 | 16;
+};
```

Comment:
Allow Boxed with 16 border radius, but only internally, not exposed outside mistica

Reply:
why no public? if you want some custom component and lives near to a DisplayCard and want the same border radius...

Reply:
we dont want that (for the moment)
And Design is working on tokenizing border radius to be configurable by brand, so making this public will make the tokenization more dificult